operatorconfig/moduleconfig/resiliency/v1.11.0/container-powerstore-node.yaml

```diff
@@ -19,7 +19,7 @@ securityContext:
   capabilities:
     add: ["SYS_ADMIN"]
   allowPrivilegeEscalation: true
-image: quay.io/dell/container-storage-modules/podmon:v1.11.0
+image: quay.io/dell/container-storage-modules/podmon:nightly
 imagePullPolicy: IfNotPresent
 args:
```

Comment:
Along with all these changes, Please do reverify and update the minimal manifests e2e test files where it contains images and etc. And make sure it aligns with the actual one present in the samples folder

Reply:
Addressed